Jenkinsfile

```diff
@@ -116,6 +138,12 @@ def python3_gpu_ut(docker_type) {
 }
 
 try {
+    stage("Purge") {
+        node('mxnetlinux') {
+            abortPreviousRunningBuilds()
```

Comment:
I think this should be combined in the first stage. I don't see value in going back to the queue(i.e., to get an executor for the next stage) after that this stage.

Reply:
Sure, I can make it a part of Sanity Test since we plan to have that stage whether it is a full build or smoke test. 
@gautamkmr @sandeep-krishnamurthy any opinions?

Reply:
LGTM.